### DISCUSSION TITLE
No `Regex.Match(ReadOnlySpan<char> input)` returning a `ValueMatch`?

### DISCUSSION THREAD

**Discussion by vc-74** [dotnet/runtime#83720] [Q&A]
`Regex` now supports taking `ReadOnlySpan<char>` as input. For instance, `bool IsMatch(ReadOnlySpan<char> input)` is the equivalent of `bool IsMatch(string input)` for a span, improving performance and reducing memory allocation.

But why is there a `ValueMatchEnumerator EnumerateMatches(ReadOnlySpan<char> input)`, similar to `MatchCollection Matches(string input)` but no `ValueMatch Match(ReadOnlySpan<char> input)` similar to `Match Match(string input)`?

**Comment by stephentoub** [comment]
https://github.com/dotnet/runtime/issues/73223

**Comment by vc-74** [comment]
Thanks @stephentoub
